test/types/tables.jl

```diff
@@ -0,0 +1,171 @@
+@testset "$TableType" for TableType in (columntable, DataFrame)
```

Comment:
`rowtable` actually doesn't work with `apply!` because we transform to a `columntable` first. So this will have to be revisited

Reply:
https://github.com/invenia/FeatureTransforms.jl/pull/93